.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,6 +12,8 @@ What types of changes does your code introduce to Graphback?
 - [ ] Documentation Update (if none of the other choices apply)
 - [ ] Other (please specify)
 
+Link to the issue (if applicable):
```

Comment:
Remove this (this has been covered already at very top of the template.

Reply:
You could add this link to the very top along with the current text https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue

Reply:
It is a good idea but this would discourage the use of [Closing keywords](https://docs.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords)

Reply:
We want it at the very top to give the reviewer context as early as possible.

Reply:
So should I modify it to ```Fixes: #``` and put it at the top so that the reviewer gets context as early as possible and we are able to use Closing Keywords?

Reply:
No. We do not want to be too opinionated with the description as there is more than one way to link a pull request (see [here](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue)).

As well as that, not every pull request is a Fix (some are features, docs etc), and not every pull request has an issue to link. This might make it confusing for people who do not have an issue to link - it might make them think they need to create one when often this is not necessary.

Best thing to do is https://github.com/aerogear/graphback/pull/2177#discussion_r504041014